eng/packages/http-client-csharp/generator/Azure.Generator/src/AzureClientPlugin.cs

```diff
@@ -46,6 +46,8 @@ public override void Configure()
     {
         base.Configure();
         AddMetadataReference(MetadataReference.CreateFromFile(typeof(Response).Assembly.Location));
+        var sharedSourceDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location)!, "Shared/Core");
```

Comment:
We shouldn't hardcode slashes, but instead use Path.Combine and pass in each directory.

Reply:
Can we just use typeof(AzureClientPlugin).Assembly.Location?

Reply:
sure

Reply:
The difference is that we don't need to call GetExecutingAssembly which uses reflection.

Reply:
updated